app/src/App.tsx

```diff
@@ -53,6 +98,11 @@ function AppToolbar() {
 }
 
 function App() {
+  const [width, setWidth] = useState(window.innerWidth);
+  useEffect(() => {
+    window.addEventListener("resize", () => setWidth(window.innerWidth));
```

Comment:
I think you usually want to return a removeEventListener callback from useEffect so that this doesn't result in the same listener getting added over and over

Reply:
good catch thanks!